### PR TITLE
Write current module version to Verbose channel.

### PR DESCRIPTION
**Pull request by NonStatic2014**
Sometimes, our customer may use an old version of AzureML PS cmdlet and cause unexpected result. We have to let them know how to use other PowerShell cmdlet to get the module number. With this change, they just need to add `-vb` in the same command line and then they can easily get it and help the investigation.

### CODE REVIEW

**Comment by markcowl** [comment]
@NonStatic2014 This is not recommended practice for finding the module version.  If you would prefer to make this part of exception handling, or include the module version int he debug stream, this would be preferable.

**Comment by NonStatic2014** [comment]
@markcowl I tried to write it into Debug channel, but it will require user do the rest of work step by step, or setup an environment variable before run the command. I'd like to use the Verbose channel to make the user experience easier for them other more commands or action.

**Comment by markcowl** [comment]
@NonStatic2014 The assembly version is not useful information in this case, you want the module version.  Please change this to be the module version, and please put this in exception handling code and in the debug stream.

File: src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServicesCmdletBase.cs [hunk 1]
```diff
@@ -138,6 +140,7 @@ protected override void StopProcessing()
             }
             catch (Exception ex)
             {
+                this.WriteVersionInfoToDebugChannel();
```
Comment (markcowl) [inline src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServicesCmdletBase.cs]: I was thinking more of adding this information to the error itself, but you could do this in a subsequent PR

**Comment by markcowl** [comment]
ON demand run here:  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1124/

LGTM once the on-demand run passes.

File: src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServicesCmdletBase.cs (hunk 1 above)
Comment (NonStatic2014) [inline src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServicesCmdletBase.cs]: Yep, that will be a bigger change: derive from ErrorRecord, put the module version in it and return that object. Let me do that when we have more information needs to be displayed during exception handling.

**Pull request merged**